contracts/partial/yToken/lendingTypes.ligo

```diff
@@ -62,13 +62,27 @@ type liquidateParams    is [@layout:comb] record [
   collateralToken       : nat;
   borrower              : address;
   amount                : nat;
+  minSeized             : nat;
+  deadline              : timestamp;
 ]
 
 type yAssetParams       is [@layout:comb] record [
   tokenId               : nat;
   amount                : nat;
 ]
 
+type yAssetParamsWithMR is [@layout:comb] record [
```

Comment:
Do we really need to rename the types?

Reply:
`yAssetParams` used for at least `WithdrawReserve`, `AccrueInterest`, and `PriceCallback` entrypoints in codebase, so i think yes